receiver/awscontainerinsightreceiver/internal/k8sapiserver/prometheus_scraper.go

```diff
@@ -52,6 +52,8 @@ var (
 		"etcd_request_duration_seconds.*",
 		"etcd_db_total_size_in_bytes.*",
 		"apiserver_storage_list_duration_seconds.*",
+		"apiserver_flowcontrol_rejected_requests_total",
```

Comment:
nit: would be good to organize these alphabetically

Reply:
updated !